tests/Doctrine/Tests/Inflector/RulesetInflectorTest.php

```diff
@@ -23,7 +22,6 @@ class RulesetInflectorTest extends TestCase
 
     public function testInflectIrregular() : void
     {
-        /** @var Substitutions|MockObject $irregular */
         $irregular = $this->createMock(Substitutions::class);
```

Comment:
Most of these annotations are unnecessaries as we have the `phpstan-phpunit` extension to make sure of the return type of `TestCase::createMock()` :)